src/http.js

```diff
@@ -0,0 +1,64 @@
+const conf = require('./conf');
+const axios = require('axios');
+const { conj } = require('./utils');
+const { stringify: stringifyQs } = require('query-string');
+
+
+function request(opts = {}, def, http = axios) {
+  return http(configure(opts, def))
+    .then(parseResponse, throwResponse);
+}
+
+
+function configure(opts, def) {
+  opts = conj(conf, opts);
+
+  def = conj(def, {
+    headers: {},
+    paramsSerializer: stringifyQs
+  });
+
+  def = conj(def, {
+    url: opts.prefix + def.url,
+    headers: conj(def.headers, {Authorization: `Token ${opts.token}`})
+  });
```

Comment:
Is there a reason why we are not just bundling this with the previous conj (on line 16)?